### ISSUE TITLE
Parcel support (the time has come)

### ISSUE THREAD

**Issue by ocoka** [wallabyjs/public#1780]
### Issue description or question

Hello, thanks again for great product, I'm curios do you have any plans to provide support for **ParcelJS** [link](https://parceljs.org) in **WallabyJS**, like with **wallabify**.

I'm for example especially escapes from **Webpack** to **ParcelJS** for my new projects since *zero config* it's really very convenient obviously, and try to prefer COG, I really get almost all for my TypeScript, VueJS, Pug, Stylus project with no configs at all, but can't figure it out how to setup **Wallaby** to run my test in browser.

As I understand correctly, to test my code in browser, I need some bundler/runtime script loader ? Right ?

**Comment by ArtemGovorov** [comment]
> As I understand correctly, to test my code in browser, I need some bundler/runtime script loader ? Right ?

You'll need [Wallaby postprocessor](https://wallabyjs.com/docs/config/postprocessor.html) that will call Parcel to build the app (incrementally). Wallaby Webpack and Browserify postprocessors are open sourced, so one may take a look at the source code (links are at the end of the referenced docs section), and create a postprocessor for Parcel.

**Comment by ocoka** [comment]
Yeah, I already saw it, but it's not something that is easy to implement, but I will try ! At glance it's not just a Parcel / Wallaby API connection, there should be some machinery for partial handling, could you please describe generic flow, maybe patterns for such plugin implementation ?!

**Comment by ArtemGovorov** [comment]
Have a look into Webpack integration at https://github.com/jeffling/wallaby-webpack/blob/master/index.js, the file has a comment describing what's going on. Basically the idea is to call Parcel API to (incrementally) compile code and pass to Wallaby: 

- compiled results of each file,
- a loader file (file with a function that will be called from `setup` and is supposed to run unit tests/entry points).
